src/shims/lock.h

```diff
@@ -542,8 +542,9 @@ DISPATCH_ALWAYS_INLINE
 static inline dispatch_once_t
 _dispatch_once_xchg_done(dispatch_once_t *pred)
 {
-#if defined(__i386__) || defined(__x86_64__)
+#if defined(__i386__) || defined(__x86_64__) || defined(__s390x__)
 	// On Intel, any load is a load-acquire, so we don't need to be fancy
+	// s390x has a strong memory model
```

Comment:
ya I think we only have s390x in Swift so I'd like to stick with that.  Thanks for the review!